### PR TITLE
feat(proxy): keep CoCo alive

### PR DESCRIPTION
**Pull request by xla**
Bumps to the link version that keeps connections alive sending keep
alive messages. Additionally introduces the ability to listen on all
network addresses to enable communication with remote hosts.

### CODE REVIEW

**Review by FintanH — APPROVED** [review]
:metal:

**Review by rudolfs — DISMISSED** [review]

File: proxy/coco/src/config.rs [hunk 1]
```diff
@@ -13,6 +13,10 @@ lazy_static::lazy_static! {
     /// Localhost binding to any available port, i.e. `127.0.0.1:0`.
     pub static ref LOCALHOST_ANY: SocketAddr =
         SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::new(127, 0, 0, 1), 0));
+
+    /// Binds to all local addresses and any available port.
```
Comment (NunoAlexandre) [inline proxy/coco/src/config.rs:17]:
```suggestion
    /// Binds to all local interfaces and any available port.
```

To keep it consistent with the RFC: https://man7.org/linux/man-pages/man7/ip.7.html

**Review by NunoAlexandre — APPROVED** [review]
:burrito: noice!

Just left a comment but leave it up to you :seedling:

**Review by kim — APPROVED** [review]

**Review by cloudhead — DISMISSED** [review]

File: proxy/coco/src/config.rs (hunk 1 above)
Comment (xla) [reply proxy/coco/src/config.rs:17]: Good catch.

**Review by rudolfs — APPROVED** [review]

**Review by FintanH — APPROVED** [review]

**Pull request merged**